app/windows/Details/Components/FilesTab.jsx

```diff
@@ -1,35 +1,62 @@
 import React from 'react'
 
+import {
+  getStorageList,
+} from '../../../api'
+
+import ResponsiveTable from '../../../components/ResponsiveTable'
+import StorageElement from '../../Storage/Components/StorageElement'
+
 /**
  * Files tab shows the objects links within a table,
  * this information includes: id, name and size
  */
-function FilesTab({ links }) {
-  return (
-    <div>
-      <h5 className='nav-group-title'>Files</h5>
-      <table>
+class FilesTab extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {}
+    this.state.formattedLinks = []
+  }
+
+  componentDidMount(){
+    // Check all the links/dags
+    const newLinks = this.props.links.map(el => {
+      return {
+        hash: el.multihash,
+        name: el.name
+      }
+    })
+
+    getStorageList(newLinks).then(list => {
+      this.setState({formattedLinks: list})
+    })
+  }
+
+  render(){
+    return (
+      <ResponsiveTable>
         <thead>
           <tr>
+            <th></th>
             <th>ID</th>
-            <th>Name</th>
             <th>Size</th>
+            <th>Files</th>
```

Comment:
Maybe we can call this `Name`, otherwise the user might expect the same behavior like `StorageList`, i.e.: this column to describe the names of its links, which is not true